.github/bin/free-disk-space.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -euo pipefail
+
+function list_installed_packages()
+{
+    apt list --installed "$1" 2>/dev/null | awk -F'/' 'NR>1{print $1}' | tr '\n' ' '
+}
+
+function free_up_disk_space_ubuntu()
+{
+    local packages=(
+        azure-cli
+        aspnetcore-*
```

Comment:
Nit: bash will attempt to expand the wildcards here too, quote them with single quotes or escape with `\`:
```
bash-5.2$ a=(R*)
bash-5.2$ echo "${a[@]}"
README.md
bash-5.2$ ls README.md 
README.md
```